src/wp-content/themes/twentytwenty/functions.php

```diff
@@ -211,8 +215,16 @@ function twentytwenty_register_scripts() {
 		wp_enqueue_script( 'comment-reply' );
 	}
 
-	wp_enqueue_script( 'twentytwenty-js', get_template_directory_uri() . '/assets/js/index.js', array(), $theme_version );
-	wp_script_add_data( 'twentytwenty-js', 'async', true );
+	wp_enqueue_script(
+		'twentytwenty-js',
+		get_template_directory_uri() . '/assets/js/index.js',
+		array(),
+		$theme_version,
+		array(
+			'in_footer' => false,
+			'strategy'  => 'defer',
```

Comment:
Just double-checking: This was `async`, but is now `defer`. Is there a chance for any unexpected behavior due to the loading order? If so, have you tested that?